src/components/DataVisuals/index.js

```diff
@@ -18,16 +17,15 @@ const responsive = {
 };
 
 const DataVisuals = ({ title, items, ...props }) => {
-  const theme = useTheme();
-  const isTablet = useMediaQuery(theme.breakpoints.only("md"));
   const classes = useStyles(props);
 
   return (
     <Section classes={{ root: classes.root }}>
       <Typography variant="h3" className={classes.title}>
         {title}
       </Typography>
-      {isTablet ? (
+
+      <Hidden smDown lgUp implementation="css">
```

Comment:
```suggestion
      <Hidden smDown implementation="css">
```

1. We only need a different design in mobile,
2. Change `showDots` prop to be true for `md` only,
3. Include `desktop` in responsive to show 3 items.